README.md

```diff
@@ -35,11 +35,11 @@
 
 
 ## Program Schedule
-- February 27, 2024, 8:00 AM Pacific Time: Info Session [Zoom meeting link](https://us06web.zoom.us/j/84811963095?pwd=11Cqzcb4T5bPoQeLhHQxqtQ46cFz6k.1 )
-- April ~~3~~ 15, 2024, 11:59 AM Pacific Time: Notebook submission deadline (GitHub pull request)
-- April 14, 2024: Announcement of [winners](VLSI24/README.md)
-- April 17-21, 2024: Register for the conference, and obtain a letter for visa application (if needed)
-- June 16-20, 2024: Attend conference
+- October 24, 2025, 8:00 AM Pacific Time: Info Session [Zoom meeting link TBD](https://us06web.zoom.us/j/84811963095?pwd=11Cqzcb4T5bPoQeLhHQxqtQ46cFz6k.1 )
```

Comment:
I'll be traveling tat day. Should we skip the info session this year? Not sure if it's all tat valuable.

Reply:
sure